src/Tribe/Editor/Objects/Event.php

```diff
@@ -59,6 +60,16 @@ public function data( $key = null, $default = null ) {
 				'is_new_post' => true,
 			];
 
+			$startDate = tribe_get_request_var( 'tribe-start-date' );
+
+			/**
+			 * Grabs the tribe-start-date query param from the url 
+			 * if it exists then adds it to the global window object.
+			 */
+			if ( $startDate && strtotime( $startDate ) ) {
+				$this->data['tribe_start_date'] = Dates::build_date_object( $startDate )->format( Dates::DBDATEFORMAT );
+			}
```

Comment:
This is close, but best to follow the implementation that we used in the classic editor (see https://github.com/the-events-calendar/the-events-calendar/blob/master/src/Tribe/Admin/Event_Meta_Box.php#L96).

Reply:
Thanks @paulmskim I've modified this to use a similar implementation to the classic editor.